public/locales/en.json

```diff
@@ -293,6 +294,7 @@
   "utilization": "Utilization",
   "veSdl": "veSDL",
   "veSdlCalculator": "veSDL calculator",
+  "veSdlNetworkText": "Seems like you are in {{chainName}} network.\n veSDL is available on mainnet! \n Please switch to Ethereum mainnet.",
```

Comment:
nit: I'd change it to either:
Looks like you're on the {{chainName}} network. veSDL is only available on the Ethereum mainnet!
or 
Looks like you're on an unsupported network. Please switch to the Ethereum Mainnet!

Reply:
Thank you for giving me advice 🙏🏻 . I wanted to ask about it.